web/src/components/lineage/Lineage.tsx

```diff
@@ -129,6 +131,48 @@ class Lineage extends React.Component<LineageProps, LineageState> {
     })
   }
 
+  getEdges = () => {
+    const selectedPaths = this.getSelectedPaths()
+
+    return g?.edges().map(e => {
+      const isSelected = selectedPaths.some((r: any) => e.v === r[0] && e.w === r[1])
+      return Object.assign(g.edge(e), { isSelected: isSelected })
+    })
+  }
+
+  getSelectedPaths = () => {
+    const paths = [] as Array<[MqNode, MqNode]>
+
+    const getSuccessors = (node: any) => {
+      const successors = g?.successors(node)
+      if (successors?.length) {
+        for (let i = 0, count = node.length; i < count; i++) {
```

Comment:
Wouldn't a more standard loop iteration work fine here?
`for (let i = 0, i < node.length - 1; i++)`

Reply:
Lefted after initial version of code, and already updated, thanks